### DISCUSSION TITLE
Handling Curly Braces "}" in YAML for Arrangable Code Block Questions

### DISCUSSION THREAD

**Discussion by anantkm** [PrairieLearn/PrairieLearn#8688] [Q&A] [answered]
I recently created an "Arrangable code block" question, taking a template from an OER repository. Within this question, I have code snippets that include the curly braces }. However, I've noticed the YAML parser interprets these characters in a particular way that's causing issues.
Image: 
![Screenshot 2023-10-23 121050](https://github.com/PrairieLearn/PrairieLearn/assets/45778202/d9b7e9e2-8f04-4e92-b7b6-efba5b1252b1)

**Comment by nwalters512** [comment]
Can you clarify what you mean when you refer to a YAML parser? I don't believe there's any YAML involved in this question.

**Reply by anantkm** [reply]
Hey @nwalters512,

We need a YAML file for the above question. The sample question can be found here:https://us.prairielearn.com/pl/course_instance/136606/instance_question/320776540/

**Reply by nwalters512** [reply, marked as answer]
Ah, I see. In general, the solution to YAML parsing problems is to wrap things in quotes. That is, if the parser struggles with `}`, use `'}'` or `"}"` instead. Does that help?
